amarachukwu-agbo/hello-books
# #151748529 Implement pagination on client side

#### What does this PR do?

Add pagination to client-side components that render large amount of data

#### Description of Task to be completed?
- Add pagination component
- Modify actions and reducers for pagination action
- Group components that are related togethr
- Add jsdoc and proptype validation

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm start:server` and `npm run start:client`. Go to `localhost:8080/books`, if there are books, you should see the books with the pagination component underneath

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#151748529

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
